src/applications/mhv/secure-messaging/components/MessageThread/MessageThreadItem.jsx

```diff
@@ -42,10 +42,10 @@ const MessageThreadItem = props => {
       return `${!isRead ? 'New ' : ''}message ${
         fromMe ? 'sent' : 'received'
       } ${dateFormat(sentDate, 'MMMM D, YYYY [at] h:mm a z')}, ${
-        attachment ? 'with attachment' : ''
+        hasAttachments || attachment ? 'with attachment' : ''
       } from ${senderName}."`;
     },
-    [attachment, fromMe, isRead, senderName, sentDate],
+    [fromMe, hasAttachments, isRead, senderName, sentDate],

```

Comment:
do we still need to keep `attachments` in array of dependencies since it is being used in that `useEffect` on line 45?

Reply:
Good catch. Thanks! We still need to keep attachments.